src/pages/edge/adobe-journey-optimizer/index.md

```diff
@@ -87,20 +87,6 @@ iOS
 
 <Tabs query="platform=ios&task=register"/>
 
-#### Sync the user push token with Adobe
```

Comment:
we do not need to specify any API on the index page. Hence removed the `Sync the user push token with Adobe` section